packages/test-runner-mocha/src/mochaSetup.ts

```diff
@@ -1,8 +1,9 @@
 import { styles } from './styles.js';
 
-const mocha = (window as any).mocha as BrowserMocha;
-const BaseReporter = (mocha as any).Mocha.reporters.Base;
-class SilentReporter extends BaseReporter {}
+const mocha = (window as any).mocha;
```

Comment:
Is it the removal of `as BrowserMocha` that’s making TS angry here?

Reply:
You can keep the cast fyi, should work fine